sourcegraph/jetbrains
# Add llm selection dropdown

It fixes https://github.com/sourcegraph/jetbrains/issues/384 https://github.com/sourcegraph/jetbrains/issues/387 https://github.com/sourcegraph/jetbrains/issues/389
## Test plan
Scenario 1
* Login as Pro user
* Create new chat
* User is able to change default LLM

Scenario 2
* Login as Pro user
* Open chat with already sent messages
* User see default LLM but is not able to change it

Scenario 3
* Login as Free user
* Open new chat
* User doesn't see LLM dropdown

Scenario 4
* Login as Enterprise user
* Open new chat 
* User see default LLM but is not able to change it
*
